app/src/main/java/eu/kanade/tachiyomi/source/online/english/Mangadex.kt

```diff
@@ -324,6 +326,101 @@ open class Mangadex(override val lang: String, private val internalLang: String,
         return manga
     }
 
+    fun fetchFollows(filters: FilterList) = fetchFollows(0, filters)
+
+    private fun fetchFollows(page: Int, filters: FilterList): Observable<List<Pair<SManga, Int>>> {
```

Comment:
What exactly are you trying to do so i can understand?  I think after you add follows int to Smanga this will clean this up to make it understandable.